transmission_interface/include/transmission_interface/bidirectional_position_joint_interface_provider.h

```diff
@@ -0,0 +1,48 @@
+///////////////////////////////////////////////////////////////////////////////
+// Copyright (C) 2013, PAL Robotics S.L.
```

Comment:
Minor: update copyright (year, and your name / institution); this applies for this and all the new files.